R/enum.R

```diff
@@ -41,34 +59,43 @@ enum <- function(enumValues) {
 #' @return Key under which the value is stored. If the value is not in the enum,
 #'   `NULL` is returned.
 #'
+#' @family enumeration-helpers
+#'
 #' @examples
 #'
 #' Symbol <- enum(c(Diamond = 1, Triangle = 2, Circle = 2))
 #' getEnumKey(Symbol, 1)
 #' @export
+
 getEnumKey <- function(enum, value) {
   output <- names(which(enum == value))
+
   if (length(output) == 0) {
     return(NULL)
   }
+
   return(output)
 }
 
 #' Get enum values
 #'
 #' @description
+#'
 #' Return the value that is stored under the given key. If the key is not
 #' present, an error is thrown.
 #'
-#' @param enum The `enum` that contains the key-value pair
-#' @param key The `key` under which the value is stored
+#' @param enum The `enum` that contains the key-value pair.
+#' @param key The `key` under which the value is stored.
 #'
 #' @return Value that is assigned to `key`.
 #'
+#' @family enumeration-helpers
```

Comment:
What's the @family  keyword?

Reply:
https://roxygen2.r-lib.org/articles/rd.html

![image](https://user-images.githubusercontent.com/11330453/150768750-ccde2b4c-30aa-4513-9c56-92eace4ea273.png)